Makefile.core.mk

```diff
@@ -107,7 +107,7 @@ generate-k8s-client:
 .PHONY: verify-k8s-client
 build-k8s-client:
 	# verifying k8s client
-	@go build ./pkg/...
+	@go build ./pkg/... ./cmd/...
```

Comment:
Since this is not actually related to building the client, can we put this in a separate make target?

Reply:
Fixed in latest commit.